rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -291,8 +292,11 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 		{
 		case Qt::Key_E:
 		{
-			instruction_editor_dialog* dlg = new instruction_editor_dialog(this, pc, cpu, m_disasm.get());
-			dlg->show();
+			if (m_cpu)
```

Comment:
this changes the cpu object to m_cpu. was this intended or is this copy pasta from renaming cpu to m_cpu?

Reply:
Intended.